ding/policy/sac.py

```diff
@@ -781,6 +782,22 @@ def _forward_learn(self, data: dict) -> Dict[str, Any]:
                 running information.
         """
         loss_dict = {}
+        agent_data = default_preprocess_learn(
```

Comment:
move these modifications to another new class `SQILSACPolicy`, which extends `SACPolicy`, and add a switch for cos visualization